site/_data/i18n/authors.yaml

```
@@ -820,3 +820,14 @@ buettner:
     en: Michael Buettner
   description:
     en: Michael is a software engineer on the Chrome team.
+egsweeny:
+  title:
+    en: Elizabeth Sweeny
+  description:
+    en: Product manager for developer tools on the web platform in Chrome.
```

Can we please update to: 
```suggestion
    en: Elizabeth is a product manager working on developer tooling for Chrome Web Platform.
```